fe2-android/app/src/main/java/com/github/dedis/popstellar/utility/handler/WitnessMessageHandler.java

```diff
@@ -31,14 +35,14 @@ private WitnessMessageHandler() {
    * @param laoRepository the repository to access the LAO of the channel
    * @param channel the channel on which the message was received
    * @param senderPk the public key of the sender
-   * @param message the message that was received
+   * @param data the message that was received
    * @return true if the message cannot be processed and false otherwise
    */
   public static boolean handleWitnessMessage(
```

Comment:
The handleWitnessMessage is still using a boolean return value.

Reply:
oops, fixing it